docs/reference/cluster_manifest.md

```diff
@@ -141,10 +141,15 @@ These parameters are grouped directly under  the `spec` key in the manifest.
   configured (so you can override the operator configuration). Optional.
 
 * **enableConnectionPooler**
-  Tells the operator to create a connection pooler with a database. If this
-  field is true, a connection pooler deployment will be created even if
+  Tells the operator to create a connection pooler with a database for the master
+  service. If this field is true, a connection pooler deployment will be created even if
   `connectionPooler` section is empty. Optional, not set by default.
 
+* **enableReplicaConnectionPooler**
```

Comment:
Someone down the line in `connectionPooler` section needs to be mentioned, that this section alone will enable only a connection pooler for master service.

Reply:
In any way the difference when for a leader one can enable connection pool via flag or config section, but for replica only via flag, is somewhat mind boggling. The easiest way to make it more consistent is to prevent config section from enabling connection pooler for master and require flag for that as well.

Reply:
I totally agree. Both options only toggle able via "flag". Question this leaves is there anyone out there relying on the old logic where we remove the "pool" with the operator upgrade.